inventory/.env.sample

```diff
@@ -0,0 +1,13 @@
+SECRET_KEY=
```

Comment:
Hi @katendejericho5 , why do all your PRs have changes in the Inventory project? 
cc: @Psalmz777 / @BenjaminSsempala